core/safe_refcount.cpp

```diff
@@ -58,7 +58,6 @@
 	}
 
 _ALWAYS_INLINE_ uint32_t _atomic_conditional_increment_impl(volatile uint32_t *pw){
-
```

Comment:
That's another case of weird formatting where the prototype ends with `){`, not sure why. The macros also need fixing for the missing braces on single-line statement (commenting here as I see it, but it doesn't have to be handled in this PR).

Reply:
Seems like those happen because the macro call doesn't end with `;`. I'll fix that.